models/haskell/TermZipper.hs

```diff
@@ -0,0 +1,119 @@
+module Hopper.Internal.Core.TermZipper where
```

Comment:
meta remark: we should figure out some CI / buildability anti bit rot stuff for the models codes, because theres some cool ideas there